themes/default/content/registry/packages/azure-native-v2/from-v1-to-v2/_index.md

```diff
@@ -0,0 +1,218 @@
+
+// Create a resource group
+const resourceGroup = new resources.ResourceGroup("my-resource-group");
+
+// Create a user-assigned managed identity
+const userAssignedIdentity = new managedidentity.UserAssignedIdentity("my-user-assigned-identity", {
+    resourceGroupName: resourceGroup.name,
+});
+
+// Create a storage account that references the user-assigned managed identity
+const storageAccount = new storage.StorageAccount("mystorageaccount", {
+    resourceGroupName: resourceGroup.name,
+    kind: "StorageV2",
+    location: resourceGroup.location,
+    identity: {
+        type: "UserAssigned",
+        userAssignedIdentities: [userAssignedIdentity.id],
+    },
+    sku: {
+        name: "Standard_LRS",
+    },
+});
+```
+
+Before this change, the identity block would need to use an apply as follows:
+
+```
+user_assigned_identity.id.apply(lambda id: {id: {}})
+```
+
+#### Title Case Changes
+
+All resource names are now consistently in title case, starting with an uppercase letter. The previous lowercase resource names are aliased so all Pulumi state is backwards-compatible. While functions are not aliased, resolving the name discrepancy is as simple as updating the function names. For instance, in the TypeScript SDK, `azure-native.aadiam.getazureADMetric` changed to `azure-native.aadiam.getAzureADMetric`.
+
+The complete list of affected resources and functions is [in this PR](https://github.com/pulumi/pulumi-azure-native/pull/2366).
+
+
+#### MySQL and PostgreSQL Server and Flexible Server
+
+Both [Azure Database for MySQL](https://azure.microsoft.com/en-us/products/mysql) and [Azure Database for PostgreSQL](https://azure.microsoft.com/en-us/products/postgresql) are available in a `Single Server` and a `Flexible Server` variant. The `Single Server` variants are on the retirement path ([MySQL](https://learn.microsoft.com/en-us/azure/mysql/single-server/whats-happening-to-mysql-single-server), [PostgreSQL](https://learn.microsoft.com/en-us/azure/postgresql/single-server/whats-happening-to-postgresql-single-server)). Azure recommends that all new servers are created as a `Flexible Server` variant.
+
+In v2, the following resources are now associated with a `Flexible Server` variant instead of `Single Server` as they were in v1:
+- `Configuration`
+- `Database`
+- `FirewallRule`
+- `Server`
+- `PrivateEndpointConnection`
+
+Existing v1 programs upgrading to v2 using the default version will result in a change of resource type. For instance, `azure-native.dbformysql.Server` would previously have referred to a `Single Server` but will now refer to a `Flexible Server` and will result in a replacement of the resource during the next `pulumi up`. However, the properties of flexible servers are sufficiently different that, in a typed language, the program will not compile. If you would like to continue using the previous Azure API version, you may do so by using the previous explicit version.
```

Comment:
Can we name the explicit versions that have to be used, and maybe give a code example?